production/exit-node-1/application.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: exit-node-1
+  namespace: production
```

Comment:
I had problems with Applications not running in `argocd` namespace. Don't you have the same problem?